addons/escoria-core/game/core-scripts/esc/esc_action_manager.gd

```diff
@@ -540,6 +540,19 @@ func perform_inputevent_on_object(
 
 		if not player_global_pos.is_equal_approx(destination_position):
 			dont_interact = true
+			escoria.logger.warn(
+				self,
+				"Player could not reach destination coordinates %s. "  % str(destination_position) \
+					+ "Any requested action for %s will not fire." % obj.global_id
+			)
+			if "cantreach" in obj.events:
```

Comment:
Please add `cantreach` to the list of consts in the event manager class itself and reference it that way.